src/Verifier.php

```diff
@@ -215,7 +215,7 @@ public function verifyChecksumFile($checksum_file) {
             throw new VerifierException("The real path of checksum list file at \"$checksum_file\" could not be determined.");
         }
         $working_directory = dirname($absolute_path);
-        $signed_checksum_list = file_get_contents($absolute_path);
+        $signed_checksum_list = @file_get_contents($absolute_path);
```

Comment:
php 7.3 threw errors in `testChecksumFileItselfUnreadable`. This fixed that.

Reply:
This is probably okay for now (given that we ultimately validate the file contents), but it would be *so nice* if we could handle such errors more directly rather than suppressing them and failing later (which is, I assume, how this works).